### ISSUE TITLE
Messages CollectionView contentOffSet issue

### ISSUE THREAD

**Issue by Murali-Gorantla** [MessageKit/MessageKit#831]
Hi,

 When I tap any video message am going to present a AVPlayerViewController for playing videos. After playing a video when dismissed the player it back the messages screen and last message is not visible means the last message is showing below the InputBar.

Can someone suggest how can we go further? Attached Screenshot.

This issue is reproducing in MessageKit example of MessageContainerController.swift.

Below is the code added in didTapMessage method

func didTapMessage(in cell: MessageCollectionViewCell) {
        print("Message tapped")
        let videoURL = URL(string: "https://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4")
        let player = AVPlayer(url: videoURL!)
        let playerViewController = AVPlayerViewController()
        playerViewController.player = player
        self.present(playerViewController, animated: true) {
            playerViewController.player!.play()
        }
    }

![simulator screen shot - iphone 8 plus - 2018-08-30 at 10 50 33](https://user-images.githubusercontent.com/4799693/44831190-103cb380-ac43-11e8-83e7-b946b8e2f4da.png)
![simulator screen shot - iphone 8 plus - 2018-08-30 at 10 50 43](https://user-images.githubusercontent.com/4799693/44831191-10d54a00-ac43-11e8-88de-c2214d6b310e.png)

Thanks.

**Comment by SD10** [comment]
Duplicate of #755. This will be fixed in the next version, the fix is #796 on `development`. We're currently waiting on #770 to be implemented to make the release
